Source/Core/AudioCommon/OpenALStream.h

```diff
@@ -54,19 +54,23 @@
 #define FRAME_SURROUND_INT32 SURROUND_CHANNELS* SIZE_INT32
 #endif
 
-#if defined(__APPLE__)
-// OS X does not have the alext AL_FORMAT_STEREO_FLOAT32, AL_FORMAT_STEREO32,
-// AL_FORMAT_51CHN32 and AL_FORMAT_51CHN16 yet.
-#define AL_FORMAT_STEREO_FLOAT32 0
-#define AL_FORMAT_STEREO32 0
-#define AL_FORMAT_51CHN32 0
-#define AL_FORMAT_51CHN16 0
-#elif defined(_WIN32)
+// From AL_EXT_float32
+#ifndef AL_FORMAT_STEREO_FLOAT32
+#define AL_FORMAT_STEREO_FLOAT32 0x10011
```

Comment:
This looks oddly binary to me, I hope this isn't a typo and should be `0b10011` instead ;)

Also: Do the libraries understand the values correctly, even when they're not defined in the respective headers? Otherwise I don't really see the point in doing it that way...